app/src/app/home/models/metric.model.ts

```diff
@@ -43,15 +43,15 @@ export class Metric {
   }
 
   public changeDescription(): string {
-    const score = this.score();
-    if (score === 0) {
+    if (this.src === this.dst) {
       return 'same';
     }
-    const abs = Math.abs(score);
-    if (abs < 2) {
-      return `${Math.floor(abs * 100) / 1}%`;
+    if (this.dst === 0) {
+      return '';
     }
-    return `${Math.floor(abs)}x`;
+    const diff = Math.abs(this.src / this.dst - 1);
+    const speedInX = Math.round(diff * 100) / 100;
+    return speedInX >= 1 ? `${speedInX}x` : `${(speedInX * 100).toFixed(0)}%`;
```

Comment:
Not quite happy with this logic. I don't think it reports improvements correctly. For example, if `src = 500` and `dst = 1000`, it reports 50% faster. But we want it to report 2x faster.